src/App.js

```diff
@@ -1,30 +1,15 @@
 /* eslint-disable */
 import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator'
 /* eslint-enable */
 
 function App() {
```

Comment:
- [ ] The `App.js` component should be class-bassed (In the future for this project you'll see why this is important). Please fix this important issue. 🐶